README.md

```diff
@@ -176,6 +176,22 @@ In your application, at the point which the view is ready for rendering
 document.body.dispatchEvent(new Event('view-ready'))
 ```
 
+**Warning:** It is possible that your application will be ready and emit the event before the main electron process has had a chance execute the javascript in the renderer process which listens for this event.  
+
+If you are finding that the [event is not effective](https://github.com/fraserxu/electron-pdf/issues/169) and your page waits until the full timeout has occurred, then you should use `setInterval` to emit the event until it is acknowledged like so:
+
+```javascript
+  var eventEmitInterval = setInterval(function () {
+    document.dispatchEvent(new Event('ready'))
```

Comment:
I've never seen that abbreviation before and I've been coding for a very long time.  Anyway, I would say there's certainly a 0% chance it would be misinterpreted if it was typed out all the way.  I think having clearer code outweighs the benefit of typing a few less characters that are only repeated in a couple places in the code, anyway.